src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties' :
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties' :
+        for (const key of action.keysToRemove) {
+          delete state[key];
+        }
```

Comment:
Use array method forEach, it will be just one line of code

```suggestion
        keysToRemove.forEach(key => delete state[key))
```